pkg/cmd/workspace/delete.go

```diff
@@ -100,13 +109,15 @@ var DeleteCmd = &cobra.Command{
 			}
 		}
 
-		if yesFlag {
-			err := removeWorkspace(ctx, apiClient, workspace)
-			if err != nil {
-				log.Fatal(err)
-			}
-		} else {
+		if !yesFlag {
 			fmt.Println("Operation canceled.")
+		} else {
+			for _, workspace := range workspaceDeleteList {
+				err := removeWorkspace(ctx, apiClient, workspace)
```

Comment:
@shashank40 this still aborts if one workspace fails due to a problem in the `removeWorkspace` function.

You'll notice that we used `log.Fatal` in that function. That is a mistake, we should just return errors from the function at those points and then your code will be perfect.

Reply:
And one more thing, please include the workspace name in the error message. E.g. `[WORKSPACE_NAME]: ....` so it's clear which workspace failed.

Reply:
Yaaa missed that.  I have updated that in the latest commit

Reply:
![Screenshot 2024-06-06 at 09 40 49](https://github.com/daytonaio/daytona/assets/26512078/b40bfb0d-0ec5-4d6e-8e04-4d88a413230e)

> And one more thing, please include the workspace name in the error message. E.g. [WORKSPACE_NAME]: .... so it's clear which workspace failed.

Address this as well. As you can see from my screenshot, it's unclear which workspace if failing for what reason.

Reply:
removeWorkspace is now updated. It now returns errors and you will see the workspace name in the terminal

Reply:
@Tpuljak Can you please take a pull once? I see the workspace name
<img width="553" alt="image" src="https://github.com/daytonaio/daytona/assets/48802836/678268fc-7310-483c-8cd6-808e52ddedca">

Reply:
![Screenshot 2024-06-06 at 09 54 37](https://github.com/daytonaio/daytona/assets/26512078/9fa008f2-1fa6-43ac-aeb8-670a5fd9db26)

@shashank40 this is not ideal since the workspace name seems part of the error message

`log.Error(fmt.Sprintf("[%s]: %v", *workspace.Name, err))` should be much more clear.

Reply:
@shashank40 one more thing, please don't mark conversations as resolved yourself. This makes it harder for the reviewer to verify changes.
The reviewer will mark conversations as resolved once they verify the changes.

Reply:
Acknowledged the resolution part @Tpuljak .

Also this format looks cleaner. I have updated the code to show error in this format.